pmm-app/src/pmm-dbaas/components/DBCluster/DBClusterStatus/DBClusterStatus.tsx

```diff
@@ -2,31 +2,58 @@ import React, { FC, useMemo } from 'react';
 import { cx } from 'emotion';
 import { Icon, useStyles, Tooltip } from '@grafana/ui';
 import { Messages } from 'pmm-dbaas/DBaaS.messages';
+import { ProgressBar } from 'pmm-dbaas/components/ProgressBar/ProgressBar';
+import { ProgressBarStatus } from 'pmm-dbaas/components/ProgressBar/ProgressBar.types';
 import { DBClusterStatusProps } from './DBClusterStatus.types';
 import { getStyles } from './DBClusterStatus.styles';
 import { DBClusterStatus as Status } from '../DBCluster.types';
 import { STATUS_DATA_QA } from './DBClusterStatus.constants';
 
-export const DBClusterStatus: FC<DBClusterStatusProps> = ({ status, errorMessage }) => {
+export const DBClusterStatus: FC<DBClusterStatusProps> = ({
+  status,
+  message,
+  finishedSteps,
+  totalSteps,
+}) => {
   const styles = useStyles(getStyles);
   const statusError = status === Status.failed || status === Status.invalid;
+  const showProgressBar = status === Status.changing || statusError;
```

Comment:
Just to be sure, the intended here is `(status === Status.changing) || statusError`, right?

Reply:
Strict equality has higher precedence than logical OR, no need for grouping. So yes that's the intended!

Reply:
Yep, exactly 😄  👍